Header header
uint32 height
uint32 width
float32[] flow_x  
float32[] flow_y  
float32 dt        
float32 scale_factor  